tools/extra/hssi/ethernet/hssicommon.py

```diff
@@ -684,7 +685,7 @@ def eth_mode(self):
 def verify_pcie_address(pcie_address):
     m = BDF_PATTERN.match(pcie_address)
     if m is None:
-        print("Invalid pcie address foramt",pcie_address)
+        print("Invalid pcie address foramt", pcie_address)
```

Comment:
Two issues:

1. s/foramt/format/
2. Needs to be either a format string or to include a format specifier and use .format().